### ISSUE TITLE
[Odoo Apps Store] Redirect Loop on Download of purchased item

### ISSUE THREAD

**Issue by lukebranch** [odoo/odoo#14640]
<h1>Environment:</h1>
Google Chrome 54.0.2840.98 (64-bit)

<h1>Impacted versions:</h1>
Odoo online - Apps Store - Theme Store

<h1>Steps to reproduce:</h1>
<ol>
<li>purchase a theme (in my case Kingfisher from Biztech)</li>
<li>try to download the theme using the download link provided</li>
<li>you will be caught in a redirect loop due to some issues with javascript (I have not investigated this in detail yet)</li>
</ol>

<h1>Current behavior:</h1>
You will be caught in a redirect loop if you attempt to download your newly purchased module.

<h1>Expected behavior:</h1>
Your browser will download the module's zip file.

<h1>Video/Screenshot link (optional):</h1>

<img src="https://cloud.githubusercontent.com/assets/2337666/20995513/844a2da4-bd32-11e6-87a4-89b11a5aef5d.gif"/>

**Comment by lukebranch** [comment]
@Yenthe666 ,

Would you mind taking a look at this if you have time? It appears to be a pretty big bug affecting the app store that will block you from downloading purchased modules.

**Comment by Yenthe666** [comment]
@JKE-be this one is for you buddy!

**Comment by JKE-be** [comment]
Maybe @dmo-odoo can take a look ?

**Comment by JKE-be** [comment]
@lukebranch 

Could you try apps.odoo.com instead of apps.openerp.com...
Seems like the login is done for odoo.com domain and not openerp.com

From where do you have this link "openerp.com"?

**Comment by lukebranch** [comment]
@JKE-be ,

I believe I just went to odoo.com and logged in, I did not type apps.openerp.com, so i'm not sure where that URL came from. 
Let me try again later this afternoon when I have a few minutes and i'll try to reproduce it step by step.

**Comment by lukebranch** [comment]
@JKE-be ,

I'm able to reproduce some strange behaviour with the website in cache, however in incognito mode (no cache) i'm no longer able to reproduce this error.

I'll close this for now and if another issue crops up again i'll reopen it.
